documentation/meta/incidents/index.md

```diff
@@ -0,0 +1,169 @@
+you’re not certain, it is better to record it than to ignore it. Do not assume
+"someone else" is handling an incident unless they have explicitly recorded it
+using the process defined here.
+
+## How do we _identify_ incidents?
+
+Incidents are detected through various channels such as automated alerts, user
+reports, or through manual checks by maintainers. The Mini Support Rotation
+(MSR) runner makes these checks as part of their daily responsibilities.
+
+While the MSR runner monitors our communication channels closely, **any online
+maintainer who sees something that might be an incident should follow the
+incident recording process outlined here.** This does not mean that the
+reporting maintainer is inherently responsible for stabilizing the incident or
+investigating the root cause. It only means that they are responsible for making
+the initial report on the incident. Additionally, maintainers who stabilize
+services may not be the ones to perform the investigation after stabilization
+has been reached. Every incident has a [**Lead**](#how-do-we-determine-the-lead)
+who is the person responsible for providing updates on the incident and
+delegating tasks. The Lead is also responsible for hosting the retrospective
+after the incident is resolved.
+
+## How do we _classify_ incidents?
+
+Incidents are classified in two ways:
+
+- **Severity** - The impact of the incident
+- **Status** - The current state of the incident
+
+### Severity
+
+Incidents are classified using one of three levels of severity. It is important
+to note that regardless of severity, working to stabilize services (or whatever
+harm mitigation is relevant for the incident) always **takes priority over
+regular project work and MSR duties.** Once services are stabilized, any ongoing
+investigations, preventative measures, or documentation changes will be
+prioritized by the team alongside our other work.
+
+This severity scale is intentionally distinct from our GitHub priority scale.
+Any GitHub issues created to _stabilize_ a Severity 1 or Severity 2 incident are
+marked as "critical" in GitHub. Other issues are prioritized at your own
+discretion.
+
+- **Severity 1**: Critical issue affecting all/most users and all/most
+  functionalities of Openverse.
+- **Severity 2**: Major issue affecting a large number of users or critical
+  functionality of Openverse.
+- **Severity 3**: Minor issue affecting a limited number of users or
+  non-critical functionality of Openverse.
+
+### Status
+
+- **Stabilization pending** - The incident is actively disrupting service.
+- **Stabilized** - The incident is no longer directly disrupting service. Root
+  causes may still be unknown.
+- **Under investigation** - The root causes of the incident are being
+  researched. Stabilized but not yet resolved.
+- **Resolved** - The incident is stable, and long-term fixes to prevent future
+  occurrences or other mitigations have been identified.
```

Comment:
Can you clarify if the resolution of an incident only involves _identifying_ mitigations rather than implementing them? So far we've only resolved incidents after we've implemented the fixes, I think, rather than just identifying them. My perception of that could be wrong though, just thinking of the API response time incident and the one this incident response plan grew out of (https://github.com/WordPress/openverse/issues/2060).

I'm not necessarily objecting to that being the case, I just wanted to clarify if that's an intentional difference from the ad-hoc process we've followed so far. This also seems to contradict the later section about process rationale which says that resolved is "Complete the immediately necessary work to prevent this issue from recurring in the future.".

Though I think maybe my confusion is coming from the past tense of this status "resolved" rather than "resolve". Is there an intermediate status, "resolving", before "resolved"? I might also be mixing up the difference between the incident status and the status of work related to the incident. Perhaps they're separate and maybe "resolved" isn't the best word. Maybe something like "investigated" instead?

I see this as potentially similar to the distinction between "shipped" and "success" in our project statuses; or just a misunderstanding on my part of how the work to implement mitigations for an incident is related to the incident status.

Reply:
Oh thank you for pointing this out - I think I missed this in the changing of the states! I think `Resolved` should represent that the mitigations/fixes have been _implemented_ to prevent this from coming up in the future, but there may still be extant issues that would assist future disruptions (like new logging/metrics capture). I'll reword this hear to clarify a bit.